pom.xml

```diff
@@ -70,8 +70,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.8.1</version>
         <configuration>
-          <source>${maven.compiler.source}</source>
-          <target>${maven.compiler.target}</target>
```

Comment:
Moving away from variables here?

Reply:
I think we probably want to keep the vars so we don't have to traverse as far for the config